swatch-system-conduit/src/main/java/org/candlepin/subscriptions/conduit/inventory/InventoryService.java

```diff
@@ -120,7 +120,7 @@ protected CreateHostIn createHost(ConduitFacts facts, OffsetDateTime syncTimesta
   private SystemProfile createSystemProfile(ConduitFacts facts) {
     SystemProfile systemProfile = new SystemProfile();
     if (facts.getOsName() != null) {
-      systemProfile.setOperatingSystem(operatingSystemName(facts.getOsName()));
+      systemProfile.setOperatingSystem(operatingSystem(facts));
     }
     systemProfile.setOsRelease(facts.getOsVersion());
```

Comment:
@kflahert @kahowell I'm wondering if we should remove the OS release since os_release is deprecated, and the Major and minor would be giving the same information.

Reply:
Let's leave it be for now; seems puptoo populates both and I couldn't find any plans to drop it. @kruai, please let us know if you feel otherwise.

Reply:
@kahowell @ksynvrit We do plan to drop `os_release`, but we can't until everyone stops using it. The source of truth for OS information is the `operating_system` field.